spacy/tests/test_parallel.py

```diff
@@ -0,0 +1,238 @@
+import os
+from time import time
+import pytest
+import srsly
+from spacy.cli._util import load_project_config
+from spacy.cli.project.run import project_run
+from .util import make_tempdir
+
+
```

Comment:
Could you add descriptions to the docstrings describing the corresponding tests briefly?